homeassistant/components/recorder/migration.py

```diff
@@ -206,14 +206,34 @@ def _add_columns(engine, table_name, columns_def):
 
 def _modify_columns(engine, table_name, columns_def):
     """Modify columns in a table."""
+    if engine.dialect.name == "sqlite":
+        _LOGGER.warning(
```

Comment:
I don't think this should be debug level since it's only going to generate GitHub issue that nobody will be able to do anything about.  Since SQLite users will already have the short column it won't be a new problem for them so it's not something they need to be warned about.

Reply:
No, this will never run! We skip sqlite, when we upgrade:
```
if engine.dialect.name != "sqlite":
    _modify_columns(engine, "events", ["event_type VARCHAR(64)"])
```
This is for a later situation, when somebody calls this function for sqlite, and forget, that he shouldn't call this.
But!
Maybe we can skip silently sqlite in `_modify_columns`, because it [doesn't care about column length](https://www.sqlite.org/datatype3.html#affinity_name_examples)? - thanks @B5r1oJ0A9G

Reply:
I modified it, now we can call `_modify_columns` always, it will skip sqlite silently, only a debug level log!